README.md

```diff
@@ -1 +1,44 @@
-# mobile-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> The project is to create the mobile version of my Portfolio.
```

Comment:
- [ ] Please update your Readme, currently some headings are missing such as  

```  ##  Getting Started ```
- [ ] Your README looks great, good job so far 👍🏻  However  it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc. Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.